config/schema/og.schema.yml

```diff
@@ -110,6 +110,9 @@ og.og_role.*:
     group_bundle:
       type: string
       label: 'Group bundle'
+    is_admin:
```

Comment:
We could also adopt fromthing like core's RoleStorage, a `isPermissionInRoles` method. That would check each role for x permission or admin.